src/client/themes/styled.d.ts

```diff
@@ -134,18 +147,17 @@ declare module 'styled-components' {
         };
       };
 
-      walletButton: {
-        background: string;
-        color: string;
-      };
+      // walletButton: {
+      //   background: string;
+      //   color: string;
+      // };
```

Comment:
Remove?

Reply:
Removed in 37090db0d7efc2706a2f90189ac28479f41cded6